bindings/python/test/flight/test_profile.py

```diff
@@ -32,96 +33,79 @@
-
-################################################################################################################################################################
+    def dynamic_provider_generator (instant: Instant):
+        return Transform.identity(instant)
 
-# def test_flight_profile_comparators ():
+    return Profile(DynamicProvider(dynamic_provider_generator), Frame.GCRF())
 
 ################################################################################################################################################################
 
-def test_flight_profile_undefined ():
+class TestProfile:
 
-    profile: Profile = Profile.undefined()
+    def test_constructors (self, profile: Profile):
 
-    assert profile is not None
-    assert isinstance(profile, Profile)
-    assert profile.is_defined() is False
+        assert profile is not None
+        assert isinstance(profile, Profile)
 
-################################################################################################################################################################
+    def test_get_state_at (self, profile: Profile, instant: Instant):
 
-# def test_flight_profile_inertial_pointing ():
+        state: State = profile.get_state_at(instant)
 
-#     quaternion: Quaternion = Quaternion([0.0, 0.0, 0.0, 1.0], Quaternion.Format.XYZS)
+        assert state is not None
+        assert isinstance(state, State)
+        state.is_defined()
 
-#     trajectory: Trajectory = Trajectory.
+    def test_get_states_at (self, profile: Profile, instant: Instant):
 
-#     profile: Profile = Profile.inertial_pointing(quaternion, trajectory)
+        states = profile.get_states_at([instant, instant])
 
-#     assert profile is not None
-#     assert isinstance(profile, Profile)
-#     assert profile.is_defined()
+        assert states is not None
 
-################################################################################################################################################################
+    def test_get_axes_at (self, profile: Profile, instant: Instant):
 
-# def test_flight_profile_nadir_pointing ():
+        axes = profile.get_axes_at(instant)
 
-#     quaternion: Quaternion = Quaternion([0.0, 0.0, 0.0, 1.0], Quaternion.Format.XYZS)
+        assert axes is not None
+        assert isinstance(axes, Axes)
 
-#     trajectory: Trajectory = Trajectory.
+    def test_undefined (self):
 
-#     profile: Profile = Profile.inertial_pointing(quaternion, trajectory)
+        profile: Profile = Profile.undefined()
 
-#     assert profile is not None
-#     assert isinstance(profile, Profile)
-#     assert profile.is_defined()
+        assert profile is not None
+        assert isinstance(profile, Profile)
+        assert profile.is_defined() is False
 
-################################################################################################################################################################
+    # def test_inertial_pointing (self):
```

Comment:
Unrelated to these changes, but should these be commented out / skipped?

Reply:
Fixed!